module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,57 @@
+#include "compression.hpp"
+
+#include <iostream>
+
+void printMap(const std::array<std::array<uint8_t, width>, height>& bitmap) {
+
+    for (const auto& arr : bitmap) {
+        for (const auto& el : arr) {
+            if (el <= ' ')
```

Comment:
why?

Reply:
I think this is because some values in array are nonprintable (below 32 in ASCII table).

Reply:
So good option here is created for example `constexpr char BeginingOfNonPrintableCharacter = ' ';` and if statement will be look 
```
if (el <= BeginingOfNonPrintableCharacter ) {

}
```